drools-bom/pom.xml

```diff
@@ -1086,6 +1086,12 @@
       <!--REST-->
 
       <!--drools-wb distribution-->
+      <dependency>
+        <groupId>org.drools</groupId>
+        <artifactId>drools-wb-ui</artifactId>
```

Comment:
Could you please explain what's the purpose of this new artifact?

Reply:
answered: the reason is build process - ui module contains client code, webapp backend + descriptors.